Formula/aris.rb

```diff
@@ -0,0 +1,29 @@
+class Aris < Formula
+  desc "Logical proof program"
+  homepage "https://github.com/kovzol/aris"
+  url "https://github.com/kovzol/aris/archive/refs/tags/2.2-15042023.tar.gz"
+  sha256 "7c44f0d53c1c2ab535653159f9aac47c9614426756d6f7c35bea19024331d3b1"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "cairo"
+  depends_on "coreutils"
+  depends_on "gtk+3"
+  depends_on "pango"
+  depends_on "pkg-config"
+  uses_from_macos "libxml2"
+  
+  on_linux do
+    depends_on "libxml2"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    system "#{bin}/aris", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I fixed this, thank you for pointing to the issue.
There is still one problem: `GitHub repository not notable enough`, and this is indeed so, the project is not yet well-known at GitHub. But, in fact, Aris has just recently been brought to GitHub, it was formerly maintained at GNU's homepage at https://www.gnu.org/software/aris/ and at http://savannah.gnu.org/projects/aris/, but it is truly an official part of the GNU project. So it might be considered as "notable enough", just because of part of GNU. In addition, its snapcraft bundle (for Linux), available at https://snapcraft.io/aris, has already more than 80 registered installations (it is linearly increasing since its debut March 2022).